chapters/copd/main.tex

```diff
@@ -849,7 +849,7 @@ \subsection{Moving arrivals between clusters}\label{subsec:moving}
 so does each cluster's \(p_i\)), but their arrival rates are altered according
 to some transfer proportion. Consider two clusters indexed at \(l\) and \(m\),
 and their respective arrival rates, \(\lambda_l\), \(\lambda_m\). Let \(\delta
-\in [0, 1]\) denote the proportion of arrivals to be moved from cluster \(l\) to
+\in [0, 1)\) denote the proportion of arrivals to be moved from cluster \(l\) to
```

Comment:
We did briefly discuss this (in the summer, I think?) and it came from a technical issue where `ciw` cannot have no arrivals from a customer class, and its class naming system not allowing us to have classes 0, 1 and 3, without 2, say. A workaround would be to have a dictionary of cluster label to class that could be recovered (although this could be awkward given how the results are generated and recorded)

In any case, we needed to get the ball rolling with the results and I just put this on the back burner. Running that whole experiment only takes a day (or thereabouts) so we can always come back and pick up that case.

Reply:
No that's fine, not worth changing completely valid as is. I'd just forgotten our conversation :) :+1: